### ISSUE TITLE
Provide visual feedback when Podman machine is starting

### ISSUE THREAD

**Issue by deboer-tim** [containers/podman-desktop#94]
### Is your enhancement related to a problem? Please describe

Podman machine can take 15s to start, even on a high-end machine. In the UI when I click Start, the status changes within a couple seconds but then there are no containers or images until it actually starts. The first couple times I thought the UI wasn't connecting properly or maybe the start hadn't worked.

### Describe the solution you'd like

We need a 'starting' state, some kind of progress, or something to let the user know that they're still waiting on Podman.

**Comment by benoitf** [comment]
Yes issue is on podman side that has no 'intermediate' step as well
https://github.com/containers/podman/issues/13711#issuecomment-1095204049

But yes, will add something on Desktop side as well

**Comment by deboer-tim** [comment]
I hit this again yesterday and almost opened another issue - I hit the button and nothing changed for several seconds, so I didn't know if it was doing anything or if the click hadn't registered.

I think we're going to find several cases where kicking off a task doesn't have progress, or maybe the command finishes and we need to wait for a status update elsewhere. In this case even disabling the button for a few seconds would have given me enough feedback that it was doing something, ideally maybe an indication that it's doing something in the background and re-enabling the button on completion, failure, or timeout.

**Comment by benoitf** [comment]
~~Hi @deboer-tim it’s slow as the exe is unpacking the content to a temporary directory 'each time' (it's how the portable exe is done by electron-builder)~~

~~If you use the setup / https://github.com/containers/podman-desktop/releases/download/v0.0.5/podman-desktop-0.0.5-setup.exe file you should have a faster experience as basically all the files are already extracted to a given directory during the install.~~

**Comment by deboer-tim** [comment]
Faster startup is good, but I opened this more for the fact that nothing happens when you click on the Start button. Even if it only takes a few seconds, it would be nice if (e.g.) the Start button disabled and status changed to 'Starting' when you clicked on it, then status changes again when it's started (and Stop button enabled) or fails/time-out.

Ditto for the reverse when stopping.

**Comment by benoitf** [comment]
I think I didn't commented the right issue, it was meant to https://github.com/containers/podman-desktop/issues/293

**Comment by slemeur** [comment]
This has been done
